docs/node-operators/migration-guides/upgrade-node.md

```diff
@@ -30,8 +30,38 @@ Ahead of the upgrade, insert the URL for your Arbitrum archive node in `YOUR_VEG
   [[Ethereum.EVMBridgeConfigs]]
     ChainID = "42161" << use this chain ID
     RPCEndpoint = "ARBITRUM_RPC" <<< set your archival node RPC endpoint here
+
+  [EvtForward.Ethereum]
+  ...
+  ...
+  [[EvtForward.EVMBridges]]
+    Level = "Info"
+    PollEventRetryDuration = "500ms"
+    MaxEthereumBlocks = 10000
+    ChainID = "42161"
 ```
 
+:::warning
+You must select a big enough value for the `MaxEthereumBlocks` because Arbitrum produces about five blocks per second. If the range is too small, your node will not keep up with the rest of the network.
+:::
+
+:::note Find blocks spam for your RPC provider
+You must check how big the span of the blocks you can use with your provider. Otherwise, your validator won't be able to validate events from the Arbitrum network.
```

Comment:
```suggestion
You must check how big the span of the blocks is that you can use with your provider. Otherwise, your validator won't be able to validate events from the Arbitrum network.
```